soh/soh/Enhancements/TimeSavers/SkipCutscene/SkipIntro.cpp

```diff
@@ -10,12 +10,12 @@ extern PlayState* gPlayState;
 extern SaveContext gSaveContext;
 }
 
-void SkipIntro_Register() {
+void RegisterSkipIntro() {
     REGISTER_VB_SHOULD(VB_PLAY_TRANSITION_CS, {
         // If we're playing rando and if starting age is adult and/or overworld spawns are shuffled we need to skip
         // the cutscene regardless of the enhancement being on.
         bool adultStart = gSaveContext.linkAge == LINK_AGE_ADULT;
-        bool shuffleEntrances = OTRGlobals::Instance->gRandoContext->GetOption(RSK_SHUFFLE_ENTRANCES).Is(true);
+        bool shuffleEntrances = Rando::Context::GetInstance()->GetOption(RSK_SHUFFLE_ENTRANCES).Is(true);
```

Comment:
Not going to block the PR over this, but I don't really think changing away from using OTRGlobals here is necessary. And with some future (potentially far future) changes I have in mind, I might even be reverting/changing this line later on anyway. No need to change it back for now though, I can worry about that later.

Reply:
Was necessary to remove dependency on OTRGlobals.h, & is consistent with other files. But thanks for the heads up